app/controllers/plants_controller.rb

```diff
@@ -0,0 +1,13 @@
+class PlantsController < ApplicationController
+    # def index
+    #     @plants = Plant.all
+    # end
+
+    def destroy
+        @plot = Plot.find(params[:plot_id])
```

Comment:
This is the right idea for this method, but a more RESTful design would be to put it in a joins controller since what you will be doing is implementing a joins table (your schema doesn't have one yet). This could go in a PlantPlotsController because you'd be deleting the joins record.